add_team_admin.py

```diff
@@ -0,0 +1,65 @@
+import synapseclient
+import json
+import logging
+
+# Initialize Synapse client
+syn = synapseclient.Synapse()
+syn.login()
+
+
+def promote_user_to_admin_with_invite(syn, team_id, user_id):
+    """
+    Invite a user to a team (if required) and promote them to admin.
```

Comment:
If I am understanding this correctly, this is proposing to promote certain users as admins (managers) on HTAN_Center_contributors and HTAN_Center_users teams. But this is not in line with what we outlined in our community management plan discussions with Lisa, where only myself and the center-specific DCC liaison are admins. We intentionally decided NOT to have center personnel as admins on center projects.

Reply:
This is just a helper function used only when first setting up the teams and promoting you admin. It is not deployed routinely

Reply:
Sure. That's fine.